mc_include.cmd

```diff
@@ -1524,34 +1541,50 @@ STOW:
 STOW_LEFT:
      if "$lefthandnoun" != "" then
           {
-               if matchre("%alltools", "$lefthandnoun") then 
+               if matchre("%tiedtools", "$lefthandnoun") then 
                     {
-                    if matchre("%tiedtools", "$lefthandnoun") then 
-                         {
-                         send tie my $lefthandnoun to my $MC_TOOLBELT_%society.type
-                         pause 0.5
-                         if "$lefthand" != "Empty" then gosub PUT_IT my $lefthandnoun in my %tool.storage
-                         }
-                    else gosub PUT_IT my $lefthandnoun in my %tool.storage
+						pause 0.5
+                        send tie my $lefthandnoun to my $MC_TOOLBELT_%society.type
+                        # pause 0.5
+                        # if ("$lefthand" != "Empty") then gosub PUT_IT my $lefthandnoun in my %tool.storage	
                     }
-               else gosub PUT_IT my $lefthandnoun in my %main.storage
+                else
+					{
+						if matchre("%alltools", "$lefthandnoun") then 
+							{
+								gosub PUT_IT my $lefthandnoun in my %tool.storage
+							}
+						else
+							{
+								gosub PUT_IT my $lefthandnoun in my %main.storage
+							}
+					}
           }
 	return
      
 STOW_RIGHT:
      if "$righthandnoun" != "" then
           {
-               if matchre("%alltools", "$righthandnoun") then 		
+               if matchre("%tiedtools", "$righthandnoun") then 		
                     {
-                    if matchre("%tiedtools", "$righthandnoun") then 
-                         {
-                         send tie my $righthandnoun to my $MC_TOOLBELT_%society.type
-                         pause 0.5
-                         if "$righthand" != "Empty" then gosub PUT_IT my $righthandnoun in my %tool.storage
-                         }
-                    else gosub PUT_IT my $righthandnoun in my %tool.storage
+                         pause 0.1
+                         gosub PUT tie my $righthandnoun to my $MC_TOOLBELT_%society.type
+                         # pause 0.5
+                         # if ("$righthand" != "Empty") then gosub PUT_IT my $righthandnoun in my %tool.storage
                     }
-               else gosub PUT_IT my $righthandnoun in my %main.storage
+                else 
+					{
+						if matchre("%alltools", "$righthandnoun") then 
+							{
+								gosub PUT_IT my $righthandnoun in my %tool.storage
+							}
+						else 
+							{
+								gosub PUT_IT my $righthandnoun in my %main.storage
+							}
+					}
+                    
+               

```

Comment:
This is where the bulk of the changes happened. This logic has been massively rewritten. Validated it works both with Alchemy and Forging toolbelts